pkg/ibmcsidriver/config_watcher.go

```diff
@@ -72,22 +71,17 @@ func (cw *ConfigWatcher) updateSubnetList(oldObj, newObj interface{}) {
 		// Env variable VPC_SUBNET_IDS will be updated only when there is
 		// non empty data and there is change in configmap
 		if newConfig != "" && (newConfig != oldConfig) {
-			VPC_SUBNET_IDS := newData.Data[ConfigmapDataKey]
-			err := os.Setenv("VPC_SUBNET_IDS", VPC_SUBNET_IDS)
+			err := os.Setenv("VPC_SUBNET_IDS", newConfig)
```

Comment:
Change the variable names `newConfig` `oldConfig` to something that is easy to understand VPC_SUBNET_IDS_NEW/OLD OR similar.